datafusion/substrait/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -200,6 +200,11 @@ async fn select_with_filter() -> Result<()> {
     roundtrip("SELECT * FROM data WHERE a > 1").await
 }
 
+#[tokio::test]
+async fn select_with_filter_sort_limit() -> Result<()> {
+    roundtrip("SELECT * FROM data WHERE a > 1 ORDER BY b ASC LIMIT 2").await
```

Comment:
It might also be worth a test for `OFFSET`

Like

```sql
    roundtrip("SELECT * FROM data WHERE a > 1 ORDER BY b ASC LIMIT 2").await
```

```
> select * from values (1), (2), (3) ORDER BY column1 LIMIT 1 offset 2;
+---------+
| column1 |
+---------+
| 3       |
+---------+
1 row(s) fetched.
Elapsed 0.012 seconds.
```

Reply:
Good idea.  I'll add.

Reply:
added